benchmark.c

```diff
@@ -167,15 +169,26 @@ __m128i runskinnyprune_epi8(int * bitmasks, int N, __m128i *x) {
 
 
 #ifdef __AVX512VBMI2__ 
```

Comment:
There are 2 #ifdef __AVX512VBMI2__ in the same macro scope in line 160 and 171 and I believe there is one #endif missing, but see this is just as a drive by comment and feel free to close.

Reply:
Nice catch. Idk why but the GitHub editor decided to be mean